tools/build_paks.py

```diff
@@ -53,135 +125,58 @@
     else:
         assert False
 
-    assert os.path.exists(unreal_editor_bin)
-    assert os.path.exists(unreal_pak_bin)
-
-    # once we know the platform, set our cooked dir
-    unreal_project_cooked_dir       = os.path.realpath(os.path.join(unreal_project_dir, "Saved", "Cooked", platform))
-    unreal_project_cooked_dir_posix = unreal_project_cooked_dir.replace(ntpath.sep, posixpath.sep)
-
-    # We do not want to use os.path.realpath(...) here, because that will resolve to the directory inside the user's Perforce workspace.
-    # Instead, we want this path to refer to the symlinked version inside the user's unreal project directory.
-    unreal_project_content_scenes_dir = os.path.join(unreal_project_content_dir, "Scenes")
-
-    # We use different strategies for setting scene_ids, depending on if we're creating symlinks or not.
-    # If we're not creating symlinks, then the user must specify args.scene_ids. If we are creating
-    # symlinks, then we get a list of candidate scene_ids from Perforce and optionally filter.
-    if args.skip_create_symlinks:
-        scene_ids = [args.scene_ids]
-    else:
-        perforce_content_scenes_dir = os.path.realpath(os.path.join(args.perforce_content_dir, "Scenes"))
-        assert os.path.exists(perforce_content_scenes_dir)
-
-        ignore_names = [".DS_Store"]
-        scene_ids = [ os.path.basename(x) for x in os.listdir(perforce_content_scenes_dir) if x not in ignore_names ]
-        assert len(scene_ids) > 0
-        if args.scene_ids is not None:
-            scene_ids = [ s for s in scene_ids if fnmatch.fnmatch(s, args.scene_ids) ]
-        assert len(scene_ids) > 0
-
-    # Create a symlink to common top-level directories
-    if not args.skip_create_symlinks:        
-        # We do not want to use os.path.realpath(...) for the values in this dictionary, because that will resolve
-        # to the directory inside the user's Perforce workspace. Instead, we want this path to refer to the symlinked
-        # version inside the user's Unreal project directory.
-        perforce_dirs_to_unreal_dirs = {
+    unreal_project_content_dir           = os.path.realpath(os.path.join(unreal_project_dir, "Content"))
+    unreal_project_cooked_dir            = os.path.realpath(os.path.join(unreal_project_dir, "Saved", "Cooked", platform))
+    unreal_project_content_scenes_dir    = os.path.realpath(os.path.join(unreal_project_content_dir, "Scenes"))
+    unreal_project_default_manifest_file = os.path.realpath(os.path.join(unreal_project_cooked_dir, "SpearSim", "Metadata", "packagestore.manifest"))
+    perforce_content_scenes_dir          = os.path.realpath(os.path.join(args.perforce_content_dir, "Scenes"))
+
+    # get asset names that are available by default
```

Comment:
I just renamed the inner variable in the list comprehension to make things a bit more concise.

```
    # get asset names that are available by default
    assert os.path.exists(unreal_project_default_manifest_file)
    with open(unreal_project_default_manifest_file) as f:
        default_manifest = json.load(f)
        default_manifest_asset_names = [ asset["Path"] for asset in default_manifest["Files"] ]
```